app/models/related_link.rb

```diff
@@ -3,7 +3,25 @@
 
 # Models a URI that is related to a work
 class RelatedLink < ApplicationRecord
+  extend T::Sig
+
+  COCINA_HASH_TYPE_NO_TITLE = T.type_alias { { type: String, access: { url: T::Array[{ value: String }] } } }
```

Comment:
How about naming these after the Cocina types that they represent?  e.g DescriptiveValue, Title, Event, etc.